cloud-agent/service/server/src/main/scala/org/hyperledger/identus/oid4vci/service/OIDCCredentialIssuerService.scala

```diff
@@ -179,7 +177,7 @@ case class OIDCCredentialIssuerServiceImpl(
   }
 
   def buildJwtVerifiableCredential(
-      issuerDid: PolluxDID,
+      issuerDid: String,
```

Comment:
As far as I know, there is a suitable type, `DID,` that should be used for the fields that look like DID in the package 
`package org.hyperledger.identus.castor.core.model.did`